README.md

```diff
@@ -26,6 +26,22 @@ D19楼长在楼栋群中公布了搬迁的大致安排。根据这份安排，
 
 D1、D14的地理位置差于D19，宿舍设施不优于D19。
 
+### 2019年7月25日
+
+更新二月十三微博上披露的致远大楼规划，将建在D19原址上。
+
+![2019072505](C:\Users\BruceChin\Documents\GitHub\WanderingD19\images\2019072505.jpg)
```

Comment:
Please use relative path (*eg. `![img1](images/2019072401.jpg)` in the original README*) instead of your absolute path so that it can be merged directly.